packages/flutter_tools/lib/src/base/config.dart

```diff
@@ -13,18 +13,20 @@ import 'utils.dart';
 /// A class to abstract configuration files.
 class Config {
   /// Constructs a new [Config] object from a file called [name] in the
-  /// current user's home directory as determined by the [Platform] and
-  /// [FileSystem].
+  /// current user's configuration directory as determined by the [Platform]
+  ///and [FileSystem]. The configuration directory defaults to $XDG_CONFIG_HOME
```

Comment:
good docs, but follow https://dart.dev/guides/language/effective-dart/documentation#do-start-doc-comments-with-a-single-sentence-summary and split off the additional details in a new paragraph.